app/src/main/java/explorer/city/com/cityexplorer/view/activity/MainActivity.kt

```diff
@@ -18,10 +18,7 @@ import java.util.concurrent.TimeUnit
 
 class MainActivity : LifecycleActivity() {
 
-    companion object {
-        val TAG: String = "MainActivity"
-    }
-
+    val TAG: String = "MainActivity"
```

Comment:
/shrug

I'm not sure what's correct, but I think if it works without the companion object its probably better.